### ISSUE TITLE
STT0076: Create a test set of stt training data of 10% percent from each department in addition to old test set.

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/stt-wav2vec2#11]
### Description 
we are currently using 0.5 percent of the total training samples for the test. this is not a well generalization of test set to evaluate the model. we will be creating a test set of 10 percent of total training samples. so that when we evaluate it takes into consideration broad spectrum of sample cases.

### Completion criteria
A  test set that represents ten percent of the training dataa and avialble in hugging face.

### Implementaion 
1. Download the stt training data repo and benchmark repo.
2. extract ten percent of training data from each department.
3. replace the portion of new benchmark with old test set.
4. push the new benchmark to hugging face.

### Subtask
- [x] get hugging face credentials and load training data and old benchmark
- [x] write a script to get data distribution stats withiin a department
- [x] write a script to filter a ten percent of data from department in a balanced manner
- [x]  filter ten percent from training dataset.
- [x] add with old test set. 
- [x] push benchmark to hf.
- [x] update the documentation of benchmark

**Comment by gangagyatso4364** [comment]
https://huggingface.co/datasets/ganga4364/stt_benchmark
